frontend/src/views/RoleView.vue

```diff
@@ -33,11 +53,37 @@
           </template>
         </ElDialog>
       </div>
-      <div class="NewLevel">
-        <ElButton type="primary" @click="redirectToLevelView()"> Create Level </ElButton>
-      </div>
     </div>
   </div>
+  <ElDialog v-model="deleteKnowledgeDialogVisible" title="Warning" width="500">
```

Comment:
Try to put this dialog in a separated component, with its functions. Something like components/DeleteKnowledgeDialog.vue